himbaechel/uarch/gatemate/config.cc

```diff
@@ -0,0 +1,269 @@
+/*
```

Comment:
this is copied from some of my least favourite ecp5 code, I think

is all the extra complexity for parsing config files needed here?

Reply:
Right, there is no need for this since we do not support reading